atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/ClientOptionsConstants.java

```diff
@@ -16,13 +16,11 @@
 
 package com.palantir.atlasdb.http.v2;
 
-import com.google.common.annotations.VisibleForTesting;
 import com.palantir.conjure.java.api.config.service.HumanReadableDuration;
 
 public final class ClientOptionsConstants {
     static final HumanReadableDuration CONNECT_TIMEOUT = HumanReadableDuration.milliseconds(500);
 
-    @VisibleForTesting
```

Comment:
same

Reply:
this one is actually used in production code: `(Dialogue)ClientOptions`

Reply:
ha, fair enough